_toc.yml

```diff
@@ -0,0 +1,18 @@
+# Table of contents
+# Learn more at https://jupyterbook.org/customize/toc.html
+
+format: jb-book
+root: README
+chapters:
```

Comment:
These will all appear on the right in the ToC.
Remember to add new chapters!
You can make things more elaborate by nesting, see:
https://github.com/napari/napari-workshop-template/blob/main/napari-workshops/_toc.yml
Output:
https://napari.org/napari-workshop-template/home.html